docs/api-docs/analytics/bodl-ga4.md

```diff
@@ -0,0 +1,216 @@
+# Big Open Data Layer
+
+Big Open Data Layer (shortened as `BODL`, pronounced 'Bottle') is a global JavaScript object that allows BigCommerce to integrate with third-party analytics providers. This datalayer collects storefront data on shopper behavior and holds data needed for an analytic provider, such as GA4, to make analytic reports.  With BODL, providers can easily view and manage captured data from a standardized source. Providers can reformat the event data for their solutions.
+
+This guide demonstrates how you can integrate a BigCommerce store with Google Analytics (GA4). Using BODL, you can track when a shopper starts checkout and purchases an order. You can track events for both redirected and embedded checkout.
```

Comment:
> You can track events for both redirected and embedded checkout.

I think if we mention embedded checkout here, we should explain how it works

https://bigcommercecloud.atlassian.net/browse/KB-15839 here we started topic about it

Reply:
I added a link to the embedded checkout tutorial (I'm not sure if there was anything specifically done to support embedded checkout)